<?xml version="1.0"?>
<launch>
  <node pkg="dynamic_obstacle_prediction" type="dynamic_obstacle_predictor_node" name="dynamic_obstacle_predictor_node" output="screen">
    <remap from="/markers_in" to="/cluster_markers" />
    <remap from="/markers_out" to="/predicted_markers" />
    <param name="prediction_dt" value="2.0"/>
    <param name="measurement_noise" value="5.0"/>
    <param name="process_noise" value="25.0"/>
    <param name="publish_rate" value="50.0"/>
  </node>
</launch>
